### PR TITLE
Add/admin pg

### PR DESCRIPTION
**Pull request by j-chimienti**
### Users view
screen shot 2018-12-05 at 2 43 14 pm

screen shot 2018-12-05 at 2 43 25 pm

note: to get user latest level I am grabbing currentPath from stats collection, in playeraccount, the maxLevel doesn't seem to be updating

### Levels View

screen shot 2018-12-05 at 2 43 51 pm
screen shot 2018-12-05 at 2 43 34 pm

note: levels stats gets updated on select changes

### CODE REVIEW

File: ui/src/components/KonamiCode.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="key-detection">
+  </div>
+</template>
+
+<script>
+
+const secretCode = 'mathbotadmin'
```
Comment (j-chimienti) [inline ui/src/components/KonamiCode.vue]: type secretCode to navigate to admin page

File: ui/src/services/LevelControl.js [hunk 2]
```diff
@@ -275,6 +284,10 @@ class LevelControl extends Ws {
         case 'pathAndContinent':
           this._setContinent(res, true)
           break
+
+        case 'stats':
+          this._handleUnlockAllLevels(res)
```
Comment (j-chimienti) [inline ui/src/services/LevelControl.js]: fixme: save stats

File: app/actors/messages/admin/AdminEvent.scala [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+import play.api.libs.json._
+
+object AdminEvent {
+
+  val titleLabel = "title"
+  val dateLabel = "date"
+  val descriptionLabel = "description"
+  val linksLabel = "links"
+  // val _idLabel = "_id"
```
Comment (j-chimienti) [inline app/actors/messages/admin/AdminEvent.scala]: wanted to ref _id field but had issues so created id field

**Comment by j-chimienti** [comment]
### Levels Update

color code by star sytem
make level id more readable

screen shot 2018-12-22 at 10 28 36 am



TODO:

ungroup star system, place in sequential order

**Comment by j-chimienti** [comment]
### Levels

group by star system
format table

screen shot 2018-12-24 at 12 05 09 pm

### Users


screen shot 2018-12-24 at 12 04 13 pm

cumulative new account table grouped by months



screen shot 2018-12-24 at 12 06 21 pm

**Comment by giftedprimate** [comment]
@j-chimienti Is this ready for merge?

**Comment by j-chimienti** [comment]
Believe so, I could use some feedback on components / functionality.

For most data, isAdmin users are filtered from stats like user count and level stats (besides active user count as aggregate was too expensive to run in prod)

**todo:**


1. update graph data in real time, as data fetched on an interval
1. When filter component updates, update components. Atm, only signups per day graph handles the change and updates its axis. I would also like the summary components to reload data, and signups per month if range > 2 months
1. save stats on unlock levels


screen shot 2018-12-28 at 3 47 02 pm

screen shot 2018-12-28 at 3 47 44 pm

**Comment by giftedprimate** [comment]
Awesome, I'll merge it in this afternoon and get right back to you!

Thanks!

**Pull request merged**